cucumber/api/features/change_password.feature

```diff
@@ -10,14 +10,43 @@ Feature: Change the password of a role
 
   Scenario: With basic authentication, users can update their own password using the current password.
 
-    Given I set the password for "alice" to "my-password"
-    When I successfully PUT "/authn/cucumber/password" with username "alice" and password "my-password" and plain text body "new-password"
-    Then I can GET "/authn/cucumber/login" with username "alice" and password "new-password"
+    Given I set the password for "alice" to "My-Password1"
+    When I successfully PUT "/authn/cucumber/password" with username "alice" and password "My-Password1" and plain text body "New-Password1"
+    Then I can GET "/authn/cucumber/login" with username "alice" and password "New-Password1"
 
   Scenario: With basic authentication, users can update their own password using the current API key.
 
-    When I successfully PUT "/authn/cucumber/password" with username "alice" and password ":alice_api_key" and plain text body "new-password"
-    Then I can GET "/authn/cucumber/login" with username "alice" and password "new-password"
+    When I successfully PUT "/authn/cucumber/password" with username "alice" and password ":alice_api_key" and plain text body "New-Password1"
+    Then I can GET "/authn/cucumber/login" with username "alice" and password "New-Password1"
+
+  Scenario: Succeed to set password applying to password complexity.
+
+    # 128 characters long (Requires: At least 12 characters and maximum of 128)
+    Given I set the password for "alice" to "My-Password1"
+    When I successfully PUT "/authn/cucumber/password" with username "alice" and password "My-Password1" and plain text body "My Password134567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678"
+    Then I can GET "/authn/cucumber/login" with username "alice" and password "My Password134567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678"
+
+    # with acceptable escape special characters (/)
+    Given I set the password for "alice" to "My-Password1"
+    When I successfully PUT "/authn/cucumber/password" with username "alice" and password "My-Password1" and plain text body "NewPassword1/"
+    Then I can GET "/authn/cucumber/login" with username "alice" and password "NewPassword1/"
+
+    # with mixed languages (Russian and French)
+    Given I set the password for "alice" to "My-Password1"
+    When I successfully PUT "/authn/cucumber/password" with username "alice" and password "My-Password1" and plain text body "New!Password1БбИиàâæ"
+    Then I can GET "/authn/cucumber/login" with username "alice" and password "New!Password1БбИиàâæ"
+
+  Scenario: Fail to set password not applying to password complexity.
```

Comment:
Same here. each set of `Given-When-Then` should have its own scenario

Reply:
Remove tests covered in UT and left only one in integration.